src/ember-app.js

```diff
@@ -450,6 +450,8 @@ function createShoebox(doc, fastbootInfo) {
 
     let value = shoebox[key];
     let textValue = JSON.stringify(value);
+    if (textValue === undefined) { continue; }
```

Comment:
Hmm, couldn't we use some static sigil value for undefined and avoid the issue either way?

```js
const UNDEFINED_VALUE = 'c1ba4d8b-9a14-4c06-8cd9-3eb482a99820';

//...snip...
if (textValue === undefined) { textValue = UNDEFINED_VALUE }
```

Then reverse it on reading...